src/Extension/Converters/DeprecationsReportConverter.php

```diff
@@ -0,0 +1,102 @@
+<?php
+
+/**
+ * Copyright © 2019-present Spryker Systems GmbH. All rights reserved.
+ * Use of this software requires acceptance of the Evaluation License Agreement. See LICENSE file.
+ */
+
+namespace SprykerSdk\Sdk\Extension\Converters;
+
+use SprykerSdk\Sdk\Core\Appplication\Dto\Violation\Violation;
+use SprykerSdk\Sdk\Core\Appplication\Dto\Violation\ViolationReport;
+use SprykerSdk\Sdk\Core\Appplication\Violation\AbstractViolationConverter;
+use SprykerSdk\SdkContracts\Violation\ViolationReportInterface;
+
+class DeprecationsReportConverter extends AbstractViolationConverter
+{
+    /**
+     * @var string
+     */
+    protected string $fileName;
+
+    /**
+     * @var string
+     */
+    protected string $producer;
+
+    /**
+     * @param array $configuration
+     *
+     * @return void
+     */
+    public function configure(array $configuration): void
+    {
+        $this->fileName = $configuration['input_file'];
```

Comment:
Please, cast to string to avoid type error.

Reply:
Fixed